packages/docs/developer-resources/integrations/listings.md

```diff
@@ -0,0 +1,123 @@
+# Listing Guide
+If you are a digital asset exchange or ranking site, this guide will help you run a node and audit your setup. 
+
+## Prerequisites
+This guide assumes:
+
+* You are a listing site like a digital-asset exchange (e.g. Coinbase, Binance, Kraken, etc.) or a ranking site (e.g. CMC, dAppRadar, etc.).
+* You are going to be running your own Celo blockchain node (either on your own hardware or on a cloud provider infrastructure).
+* You want to list CELO (previously cGLD)  and cUSD on your exchange or ranking site and you want to find useful information that is relevant for listing those digital assets.
+* You want to find relevant resources for brand assets that will visually aid in those listings.
+
+## Support
+If you have any questions or need assistance with these instructions, please contact cLabs or ask in the #exchanges channel on [Celo’s Discord server](https://chat.celo.org/) . Remember that Discord is a public channel: never disclose recovery phrases (also known as backup keys, or mnemonics), private keys, unsanitized log output, or personal information.
```

Comment:
```suggestion
If you have any questions or need assistance with these instructions, please contact cLabs or ask in the #exchanges channel on [Celo’s Discord server](https://chat.celo.org/). Remember that Discord is a public channel: never disclose recovery phrases (also known as backup keys, or mnemonics), private keys, unsanitized log output, or personal information.
```

Reply:
maybe include how they might be able to connect with cLabs in a private channel? My hunch is some listing services may have questions around security that may not be suitable for a public channel

Reply:
Sounds good. Any suggestions on what private channel to use? Do we have a support email for those things?